virt/esx/esx.py

```diff
@@ -310,7 +312,7 @@ def getHostGuestMapping(self):
                     # Stopped machine doesn't have any uuid
                     if vm.uuid is not None:
                         l.append(vm.uuid)
-                mapping[host.uuid] = l
+                mapping[host.name] = l
```

Comment:
What if he simply replace the mapping, and send the hostname for the HOST mapped to VMs UUID?